spec/spec_helper.rb

```diff
@@ -114,11 +115,23 @@ class Pet < ActiveRecord::Base
   col :home_address, :type => :text
   if ENV['DB'] == 'postgresql'
     col :tsntz, :type => 'timestamp without time zone'
+  else
+    add_index :name, :unique => true
+  end
+end
+if ENV['DB'] == 'postgresql'
+  begin
+    Pet.connection.execute("ALTER TABLE pets DROP CONSTRAINT IF EXISTS unique_name")
+  rescue => e
+    puts e.inspect
   end
-  add_index :name, :unique => true
 end
+
 Pet.auto_upgrade!
 
+if ENV['DB'] == 'postgresql'
```

Comment:
OK, I've got hstore support in, but it's very rough at the moment.  Need to clean it up - the problem is that the SQL can't be memoized if there are hstore columns (due to needing differing functions depending on whether or not `:eager_nullify` is set, and the column values if it is).